src/components/Editor/Runners/HtmlRunner/HtmlRunner.jsx

```diff
@@ -327,7 +326,7 @@ function HtmlRunner() {
                   "output.preview",
                 )}`}</span>
               </Tab>
-              {!!!isEmbedded && !!!webComponent && (
+              {!!!isEmbedded && (
```

Comment:
Slightly tangentially, I've not seen `!!!` before - surely this evaluates to just `!`. Is there a reason we're using it here?

Reply:
@conorriches

> surely this evaluates to just `!`

You are correct, but only if the value of `isEmbedded` was guaranteed to always be `true` or `false`.

I think this is a variation of the `!!x` pattern. The latter converts `null` and `undefined` to `false`. Thus `!!!x` that's being used here converts `null` and `undefined` to `true`. Does that make sense?

As an aside, I'm not a big fan of this pattern, because I think it's often used *unecessarily* to be defensive against unexpected values and it makes the code hard to read. I think a better approach is to ensure that the value *never* gets set to `null` or `undefined` in the first place.